subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/DefaultGradleDistribution.java

```diff
@@ -146,7 +146,7 @@ public boolean isToolingApiLoggingInEmbeddedModeSupported() {
 
     public VersionNumber getArtifactCacheLayoutVersion() {
         if (isSameOrNewer("4.4-rc-1")) {
-            return VersionNumber.parse("2.32");
+            return VersionNumber.parse("2.33");
```

Comment:
⭕️  Why did we bump this again if 4.4 isn't out yet?

Reply:
I followed the pattern that Adam uses. Even during development versions are bumped, because there's a risk that you have a local version that is not compatible (it happened to me once).